libs/Microsoft.MixedReality.WebRTC.Native/src/media/audio_transceiver.h

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "callback.h"
+#include "interop_api.h"
+#include "media/local_audio_track.h"
+#include "media/remote_audio_track.h"
+#include "media/transceiver.h"
+#include "refptr.h"
+
+namespace Microsoft::MixedReality::WebRTC {
+
+class PeerConnection;
+
+/// Transceiver containing audio tracks.
+class AudioTransceiver : public Transceiver {

```

Comment:
This and VideoTransceiver have the same exact interface, any chance that we can have just one Transceiver class that operated with media tracks, and do the checking at runtime? I don't feel like type-safety is a huge concern here - the chances of mixing audio and video by mistake seem low. If it is, we could make a template and instantiate it once for audio and once for video, so at least we save the copy&paste.

Reply:
Fixed transceivers.